packages/vue/tsconfig.json

```diff
@@ -0,0 +1,16 @@
+{
+  "compilerOptions": {
+    "target": "ESNext",
```

Comment:
Are we planning on only supporting latest modern browsers with this SDK?

Reply:
Good question! I looked up Vue's recommended [tsconfig](https://github.com/vuejs/tsconfig/blob/22e15e3947c8ce1d9c5992229da0351c464fb9e8/tsconfig.json#L56) and saw they use `ESNext`. I updated it to ES2020 to be more explicit 👍🏼